packages/ui/src/components/va-button/VaButton.vue

```diff
@@ -170,7 +170,7 @@ export default defineComponent({
     }))
 
     // styles
-    const isLowContrastBg = computed(() => props.plain || isLightBackground(colorComputed.value, props.backgroundOpacity))
+    const isLowContrastBg = computed(() => props.plain || isLightBackground(colorComputed.value, props.backgroundOpacity, 200))
```

Comment:
```suggestion
    const isTransparentBg = computed(() => props.plain || props.backgroundOpacity < 0.5)
```

Reply:
I planned to solve this while reimplementing buttons to after-before. But ok, let's make it here :)